src/interactive/results.ts

```diff
@@ -60,18 +63,21 @@ export class Result {
             if (ed.document == this.document) {
                 ed.setDecorations(this.decoration, [{
                     hoverMessage: this.content.hoverContent,
-                    range: this.range
+                    range: this.decorationRange
                 }])
             }
         }
+    }
 
+    updateDecorationRange() {
+        this.decorationRange = new vscode.Range(this.range.end.translate(0, 9999), this.range.end.translate(0, 9999))
```

Comment:
using getter will make code clearer imho:
```ts
get decorationRange(): vscode.Range 
    return new vscode.Range(this.range.end.translate(0, 9999), this.range.end.translate(0, 9999))
}
```

Reply:
Good call. Will make that change.